src/main/java/org/wildfly/security/keystore/AtomicLoadKeyStore.java

```diff
@@ -54,6 +56,13 @@ private AtomicLoadKeyStore(AtomicLoadKeyStoreSpi keyStoreSpi, Provider provider,
     public static AtomicLoadKeyStore newInstance(final String type, final Provider provider) {
         AtomicLoadKeyStoreSpi keyStoreSpi = new AtomicLoadKeyStoreSpi(() -> KeyStore.getInstance(type, provider));
 
+        if (ElytronMessages.tls.isTraceEnabled()) {
+            ElytronMessages.tls.trace("AtomicLoadKeyStore creating:" +
```

Comment:
You should use ``tracef`` for this instead of using .isTraceEnabled().  You can then also use ``%n`` instead of ``\n``.